### PR TITLE
Google tag manager

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: cafebabel/tests/test_articles.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def test_published_article_should_display_content(client, published_article,
     assert published_article.authors[0].profile.name in response
     assert ('href="https://twitter.com/share?url=http%3A%2F%2Flocalhost%2F'
             f'en%2Farticle%2F{published_article.slug}-{published_article.id}'
-            f'%2F&text={published_article.title}&via=cafebabel_eng"'
+            f'%2F&text={published_article.title}&via={twitter_nickname}"'
```
Comment (vinyll) [inline cafebabel/tests/test_articles.py]: Tu veux garder `via=cafebabel_eng` ici

File: cafebabel/tests/test_articles.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def test_article_with_non_ascii_title(published_article):
 
 
 def test_published_article_should_display_content(client, published_article,
-                                                  user):
+                                                  user, twitter_nickname):
```
Comment (vinyll) [inline cafebabel/tests/test_articles.py]: `twitter_nickname` is not something that exists.
That breaks the tests

**Review by vinyll — CHANGES_REQUESTED** [review]

File: cafebabel/tests/test_articles.py (hunk 1 above)
Comment (johangiraud) [reply cafebabel/tests/test_articles.py]: yep, I created a new branch from master, I don't understand why I get this here. My bad probably ;-)

File: cafebabel/tests/test_articles.py (hunk 2 above)
Comment (johangiraud) [reply cafebabel/tests/test_articles.py]: same

**Review by vinyll — APPROVED** [review]

**Pull request merged**